CHANGELOG.md

```diff
@@ -1,4 +1,40 @@
 # Change Log
+## v3.6.0 (2025-06-30)
+
+### NEW MODULES
+  - azure_rm_virtualnetworkgateway_info.py: ([#1966](https://github.com/ansible-collections/azure/pull/1966))
+  - azure_rm_tag.py: ([#1943](https://github.com/ansible-collections/azure/pull/1943))
+  - azure_rm_tag_info.py: ([#1943](https://github.com/ansible-collections/azure/pull/1943))
+  - azure_rm_postgresqlflexiblebackup.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+  - azure_rm_postgresqlflexiblebackup_info.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+  - azure_rm_postgresqlflexiblevirtualendpoint.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+  - azure_rm_postgresqlflexiblevirtualendpoint_info.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+  - azure_rm_postgresqlflexibleadministrator.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+  - azure_rm_postgresqlflexibleadministrator_info.py: ([1914](https://github.com/ansible-collections/azure/pull/1914))
+
+### FEATURE ENHANCEMENT
+  - ad.sh: Cleanup ado.sh ([#1972](https://github.com/ansible-collections/azure/pull/1972))
+  - pr-pipeline.yml:
+    - Set default for pr-pipeline to latest versions (python+ansible) ([#1972](https://github.com/ansible-collections/azure/pull/1972))
+    - Restore the trigger that synchronizes the dev branch during the PR test ([#1978](https://github.com/ansible-collections/azure/pull/1978}}
+  - azure-pipelines.yml: Remove unused azure-pipelines.yml ([#1976](https://github.com/ansible-collections/azure/pull/1976))
+  - release-pipelines.yml: Remove unused release-pipelines.yml ([#1976](https://github.com/ansible-collections/azure/pull/1976))
```

Comment:
These are not cx facing changes, no need to mention in the release note.

Reply:
I see, I will update as soon as possible!